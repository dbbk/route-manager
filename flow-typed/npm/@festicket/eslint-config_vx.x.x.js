// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
