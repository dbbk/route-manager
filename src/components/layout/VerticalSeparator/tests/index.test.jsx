// @flow

import React from 'react';
import { mount } from 'enzyme';
import { VerticalSeparator } from '@festicket/react-ui-components';

describe('<VerticalSeparator />', () => {
  test('with initial props', () => {
    const tree = mount(<VerticalSeparator render={() => null} />);
    expect(tree).toMatchSnapshot();
  });
});
