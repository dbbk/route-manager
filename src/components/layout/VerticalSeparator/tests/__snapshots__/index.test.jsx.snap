// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerticalSeparator /> with initial props 1`] = `
<VerticalSeparatorComponent
  render={[Function]}
/>
`;
